src/main/java/utils/StringUtils.java

```diff
@@ -0,0 +1,12 @@
+package utils;
+
+public class StringUtils {
+
+    private StringUtils() {
+        throw new IllegalStateException("유틸 클래스는 인스턴스화할 수 없습니다.");
+    }
+
+    public static boolean isNullOrEmpty(String value) {
+        return value == null || value.isEmpty();
```

Comment:
소소한 의견이긴 한데 사용의도가 빈 문자열을 체크하는 것이므로 공백까지 확인하면 더 좋을 것 같습니다 🙏 
```suggestion
        return value == null || value.isBlank();
```